Pigeonhole_Sort/PigeonholeSort.java

```diff
@@ -0,0 +1,59 @@
+public class PigeonholeSort{
+    // Method for getting the maximum value
+    public static int getMax(int[] inputArray){ 
+                int maxValue = inputArray[0]; 
```

Comment:
Fix indentation throughout the file. There is a shortcut for most editors for auto indent.